cirq/contrib/paulistring/pauli_string_phasor.py

```diff
@@ -143,8 +139,7 @@ def try_cast_to(self,
                     desired_type: Type[T_DESIRED],
                     ext: extension.Extensions
                     ) -> Optional[T_DESIRED]:
-        if (desired_type in [ops.ExtrapolatableEffect,
-                             ops.ReversibleEffect] and
+        if (desired_type in [ops.ExtrapolatableEffect] and
```

Comment:
does this need to be "in" now?

Reply:
Acknowledged. It doesn't, but this code is going to be deleted within a week when the same thing happens to ExtrapolatableEffect so it's not worth polishing.